bigbluebutton-html5/imports/ui/components/nav-bar/container.jsx

```diff
@@ -65,5 +74,8 @@ export default withTracker(() => {
     meetingId,
     presentationTitle: meetingTitle,
     hasUnreadMessages,
+    currentUser: currentUser,
+    name:currentUser.name,
```

Comment:
if currrent user is already being passed and it has name, why send name as well?